### PR TITLE
feat: add note to mynotes, archive and trash

### LINKED ISSUE #4: feature-notes

### PR DESCRIPTION
**Pull request by Bhakti199**
Here I added api calls for add and delete note from my notes, archives and trash. #4

### CODE REVIEW

File: src/Components/ColorPalette/ColorPalette.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  let colors = [
+    "#bef264",
+    "#fca5a5",
+    "#7dd3fc",
+    "#a3a3a3",
+    "#fb7185",
+    "#c084fc",
+    "rgb(255, 191, 129)",
+    "#86efac",
+  ];
```
Comment (mariyasada) [inline src/Components/ColorPalette/ColorPalette.jsx:15]: don't mix rgb with hex codes

File: src/Components/NoteEditor/NoteEditor.jsx [hunk 2]
```diff
@@ -1,44 +1,158 @@
...
-          <select className="select-section">
+
+          <select
+            className="select-section"
+            value={noteInput.priority}
+            onChange={(event) =>
+              setNoteInput((prevNote) => ({
+                ...prevNote,
+                priority: event.target.value,
+              }))
```
Comment (mariyasada) [inline src/Components/NoteEditor/NoteEditor.jsx:74]: you can make a same function for Onchange event

**Review by mariyasada — APPROVED** [review]
The code structure is good. good to merge

**Review by Manju-sree — APPROVED** [review]
Home page looks cool, please push less features in one PR.

File: src/Components/IconsContainerForNoteCard/IconsForNote.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.delete-permanent-button {
+  padding: 7px;
+  background-color: var(--blue-color);
+  /* background-color: transparent; */
```
Comment (rkapoor10) [inline src/Components/IconsContainerForNoteCard/IconsForNote.css:4]: remove this comment

**Review by rkapoor10 — APPROVED** [review]
The code is well structured looks good to merge

**Pull request merged**